h5py/_hl/filters.py

```diff
@@ -306,16 +306,23 @@ def rq_tuple(tpl, name):
 
     return plist
 
-def get_filters(plist):
-    """ Extract a dictionary of active filters from a DCPL, along with
-    their settings.
+def get_filter_name(code):
+    """
+    Return the name of the compression filter for a given filter code.
```

Comment:
```suggestion
    Return the name of the compression filter for a given filter identifier.
```

Just to align with [what HDF group call this](https://github.com/HDFGroup/hdf5_plugins/blob/master/docs/RegisteredFilterPlugins.md). I don't think we need to change it in the code, though.